update notebook names convention

joaquinhernandezg opened the pull request:
Rename notebooks m1m3 according to the format SITCOMTN-XXX_description_of_the_notebook.ipynb

Notebooks were renamed according to the technote with which part of the analysis is described or associated to, except the following notebooks:

-  ims_elevation_slew_explore.ipynb, it is a general notebook on exploring how to plot slew data, not an analysis per se. 

- SITCOM-760_Analysis.ipynb, it studies a systematic offset between the IMS and hardpoint data telemetry, and is not technote or test-specific. It was left with the name of the ticket.

- SITCOM-761_vms_io_and_plotting.ipynb is not analysis specific, but rather provides an interface to access vms data. It might be related to a vibration technote, but I could not find any. 


This was done only for m1m3 notebooks.

nsevilla commented on notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1089_strong_vibration_analysis.ipynb:
Can I ask why these files are deleted?

nsevilla commented on notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_analysis_actuators_and_actuators_movie.ipynb:
I don't see reference to this notebook or the movies in technote 92. I guess you are assuming the connection of the actuator movie to the purpose of the technote? (or maybe you checked with Bruno/Craig already)

joaquinhernandezg replied:
I asked Craig and he said those plots are not in the technote. That TN is in development though, but the ticket is closely related to the subject of SITCOMTN-092, as discussed with Craig.

nsevilla replied:
OK!

joaquinhernandezg committed:
update notebook names convention

Squashed commits
rename LVV-T232-analysis_actuators.ipynb

change underscores

Revert "update notebook names convention"

This reverts commit 9e1e5f356cfe641230bf1983602d010346efd0fc.

update notebook names convention

joaquinhernandezg commented:
@nsevilla followed your comments and used `git mv` to rename the files, which is the correct way to do it. 
Made te changes and squashed everything into a single commit.

nsevilla commented on notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_axial_breakaway_during_slew.ipynb:
Is this the correct technote? I wonder if it should be technote 82 instead. Cannot currently run the script to check. Maybe @b1quint can confirm.

joaquinhernandezg replied:
You are right. Fixed.

joaquinhernandezg committed:
Fix bad renaming

nsevilla reviewed (approved):
Looks good after comments being addressed, thank you!
